src/app/client/src/app/modules/shared-feature/components/view-all/view-all.component.html

```diff
@@ -11,7 +11,7 @@
     <div class="sb-pageSection mt-24">
         <div class="sb-pageSection-header" *ngIf="!showLoader">
             <h4 class="sb-pageSection-title m-0 mr-5">{{sectionName}}</h4>
-            <span class="sb-pageSection-count sb-label sb-label-xs sb-label-error">{{totalCount}}</span>
+            <span  class="sb-pageSection-count sb-label sb-label-xs sb-label-error">{{totalCount}}</span>
```

Comment:
why is there an extra space? And why you need to push this file as there is no changes